### PR TITLE
Rewritten tutorial + associated files

### PR DESCRIPTION
**Pull request by oatymart**
FAO @no-chris and @dietertaotesting 

Let me know your thoughts

### CODE REVIEW

File: forge/How to make a new extension 3 3/how-to-make-a-new-extension-3-3.md [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+    - 'Christophe Garcia'
+    - 'Martin Nicholson'
+tags:
+    - 'Legacy Versions:TAO 3.0'
+-->
+
+
+# How to make a new extension in TAO 3.3
+========================================
+
```
Comment (dietertaotesting) [inline forge/How to make a new extension 3 3/how-to-make-a-new-extension-3-3.md]: Don't mix "hash" notation with underline notation when making headings, hashes are preferred

File: forge/How to make a new extension 3 3/how-to-make-a-new-extension-3-3.md [hunk 2]
```diff
@@ -0,0 +1,295 @@
...
+3.  Make sure the extension has been installed (See https://hub.taocloud.org/articles/wiki/installing-a-new-extension)
+
+
+## Create the extension
+-----------------------
+1.  Log into TAO BackOffice with your admin account.
+2.  Go to "Settings" (upper right corner) > "Extensions Manager".
+3.  Click on "create new" to create your new extension.
+4.  Fill out the form and call your new extension `taoDocs`.
+    (If you wish, you can select some Samples to generate extra structure).
```
Comment (dietertaotesting) [inline forge/How to make a new extension 3 3/how-to-make-a-new-extension-3-3.md]: Can you find a more generic name? I would probaly go for something like _taoSomeMeaningfulName_

**Review by ghost — COMMENTED** [review]
Under the assumption that this tutorial is correct this is actually really good. You may want to review this article after the workshop on documentation

**Review by ghost — CHANGES_REQUESTED** [review]
Please update after doc workshop, see inline comments

**Comment by oatymart** [comment]
I agree, I wondered why `taoDocs` was used. I think the suggestion of `taoSomeMeaningfulName` is a bit long, how about I rename it `taoPlayground`?

**Comment by dietertaotesting** [comment]
`taoPlayground` could be a real thing. In fact we have a test server called `playground` and `spielplatz` on the other test server is German for `playground`, so you that might cause confusion. You could something like `taoExample|Mock|Dummy|Sample` instead. Don't worry about the length of the name though.

File: forge/How to make a new extension 3 3/how-to-make-a-new-extension-3-3.md [hunk 3]
```diff
@@ -0,0 +1,285 @@
...
+
+# How to make a new extension in TAO 3.3
+
+## Preparation
+
+1. Install TAO (3.x) (https://github.com/oat-sa/package-tao)
+2. Download `taoDevTools` extension using composer:
+    `composer require "oat-sa/extension-tao-devtools" —dev`
+3. Make sure the extension has been installed (See https://hub.taocloud.org/articles/wiki/installing-a-new-extension)
+
```
Comment (dietertaotesting) [inline forge/How to make a new extension 3 3/how-to-make-a-new-extension-3-3.md]: Link this to the article on GitHub instead, use a proper md link
https://github.com/oat-sa/taohub-articles/blob/master/forge/Wiki/installing-a-new-extension.md

**Review by ghost — COMMENTED** [review]
All screen shots refer to `taoDocs`, which is wrong for two reasons
- the extension is now called `taoExample`
- more importantly you would not have the label `taoDocs` or `taoExample` in the menu - you would have `Docs` or `Example` instead. 
I suggest you figure out how this is done (in `structures.xml`), describe and make new shots (or use https://pixlr.com/editor/ or something like that)

**Comment by oatymart** [comment]
You are right of course. I will look into editing or redoing the screenshots.

**Pull request merged**